docs/index.rst

```diff
@@ -71,22 +71,19 @@ This tool helps transform legacy configuration files templated with the atparse
 
 | :any:`CLI documentation with examples<cli_template_translate_examples>`
 
-File Provisioning
-^^^^^^^^^^^^^^^^^
+File/Directory Provisioning
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This tool helps users define the source and destination of files to be copied or linked, in the same UW YAML language used by UW drivers.
+This tool helps users define the source and destination of files to be copied or linked, or directories to be created, in the same UW YAML language used by UW drivers.
 
-| :any:`CLI documentation with examples<cli_file_copy_examples>`
+| :any:`CLI documentation with examples<cli_fs_mode>`
 
-
-There is a video demonstration of the use of the ``uw file`` tool available via YouTube.
+There is a video demonstration of the use of the ``uw fs`` tool (formerly ``uw file``) available via YouTube.
```

Comment:
I hadn't thought about the video demo. That's a bummer for it to already be outdated.

Reply:
Yes. If someone wanted to get some video-demo experience without having to think too hard about the content, they could re-record that, updating the mode name and providing a small extra section on the `makedirs` action.